modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -28,26 +28,27 @@ and which CSI features they support, such as volume snapshots and resize.
 [cols=",^v,^v,^v,^v,^v width="100%",options="header"]
 |===
 |CSI driver |CSI volume snapshots  |CSI cloning  |CSI resize |Inline ephemeral volumes
-|AWS EBS | ✅ | - | ✅| -
-|AWS EFS | - | - | -| -
+|AWS EBS | ✅ |  | ✅| 
+|AWS EFS |  |  |  | 
 ifndef::openshift-rosa[]
-|Google Compute Platform (GCP) persistent disk (PD)|  ✅|  ✅ | ✅| -
-|GCP Filestore | ✅ | - | ✅| -
+|Google Compute Platform (GCP) persistent disk (PD)|  ✅|  ✅ | ✅|  
+|GCP Filestore | ✅ |   | ✅|  
 endif::openshift-rosa[]
 ifndef::openshift-dedicated,openshift-rosa[]
-|{ibm-power-server-name} Block | - | - | ✅ | -
-|{ibm-cloud-name} Block | ✅^[3]^ | - | ✅^[3]^| -
+|{ibm-power-server-name} Block |   |   | ✅ |  
+|{ibm-cloud-name} Block | ✅^[3]^ |   | ✅^[3]^|  
 endif::openshift-dedicated,openshift-rosa[]
-|LVM Storage | ✅ | ✅ | ✅ | -
+|LVM Storage | ✅ | ✅ | ✅ |  
 ifndef::openshift-dedicated,openshift-rosa[]
-|Microsoft Azure Disk | ✅ | ✅ | ✅| -
-|Microsoft Azure Stack Hub | ✅ | ✅ | ✅| -
+|Microsoft Azure Disk | ✅ | ✅ | ✅|  
+|Microsoft Azure Stack Hub | ✅ | ✅ | ✅|  
 |Microsoft Azure File | - | - | ✅| ✅
```

Comment:
Just an FYI that if you're doing a sweep of unnecessary hyphens then this line still has some